domains/jariullah.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Personal Portfolio",
+  "repo": "https://github.com/Mohdjariullah/portfolio1",
+  "owner": {
+    "username": "mohdjariullah",
+    "email": "mohdjariullah@gmail.com"
+  },
+  "record": {
+    "CNAME": "mohdjariullah.github.io"
```

Comment:
Little content is provided on this site.

Reply:
huh? Man it is a portfolio what else do you expect

Reply:
It has your name and the site name. That's not a portfolio that's two HTML lines.

Reply:
Sir that's not my portfolio 
https://portfolio1-sigma-three.vercel.app/
This is my portfolio1

Reply:
@DEV-DIBSTER i believe this is his portfolio https://github.com/mohdjariullah/portfolio1/

Reply:
> @DEV-DIBSTER i believe this is his portfolio https://github.com/mohdjariullah/portfolio1/

Yep